x/crosschain/types/message_vote_gas_price.go

```diff
@@ -53,5 +54,8 @@ func (msg *MsgVoteGasPrice) ValidateBasic() error {
 	if msg.ChainId < 0 {
 		return cosmoserrors.Wrapf(sdkerrors.ErrInvalidChainID, "chain id (%d)", msg.ChainId)
 	}
+	if chains.IsBitcoinChain(msg.ChainId, []chains.Chain{}) && msg.PriorityFee != 0 {
```

Comment:
Can we add comment why this check specifically for Bitcoin

Reply:
description added: https://github.com/zeta-chain/node/pull/3377/commits/98a7472ac6d9abc750b72d89de04c70e93496a32